package.json

```diff
@@ -17,6 +17,7 @@
     "babel-preset-es2015": "~6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-register": "^6.9.0",
+    "clipboard": "^1.7.1",
```

Comment:
we were able to do this without extra dependency in [plugin](https://github.com/theforeman/foreman_virt_who_configure/blob/fd523b4e06dce034ad232ac7eba24d2ca597385c/app/assets/javascripts/foreman_virt_who_configure/config_copy_and_paste.js), maybe same approach could be taken here or if not, it would be great if you provide similar JS function so others could use it instead of writing `<%= javascript_tag("new Clipboard('#personal-access-token-clipboard');"); %>`